### ISSUE TITLE
Can't get it to work on macOS

### ISSUE THREAD

**Issue by michaelbaisch** [dhelonious/PipenvPath#1]
I installed PipenvPath with the Sublime's Package Control.
I set the Build System simply to "Python" and in the preferences I also specifically set where the pipenv executable is:

```yaml
"pipenv_path":
  {
      "osx": "/opt/homebrew/bin/pipenv",
  },
```

But the virtualenv of pipenv isn't used. Looking in the source I see a few `print` commands. Should those be printed in the Sublime Console? I don't see any of them. Anything I can do to further debug this or did I do something completely wrong?

**Comment by dhelonious** [comment]
Sorry for the late reply. I have not used this plugin for some time. The print statements should indeed print to the ST console, you should see something like this:
```
...
plugins reloaded
...
PipenvPath: /opt/homebrew/bin/pipenv
...
```
However, you must be in a project for PipenvPath to do something at all (this might not be entirely clear from the Readme). If you did not use a project for testing, then please try again if the plugin works.

Otherwise, I did never test this plugin on macOS. I assumed that mac uses the same file structure as linux, i. e. pipenv stores the environments in `~/.virtualenvs` and the executables are located in `~/.virtualenvs/pipenv-.../bin` where `...` is some hash. Is this path correct?

**Comment by michaelbaisch** [comment]
No worries.

Out of curiosity I added a `.sublime-project` file to my folder so sublime would recognize it as project, but there is no change. There is no `~/.virtualenvs` on my machine and `pipenv --venv` gives me something like `~/.local/share/virtualenvs/<project_name>-<hash>`.

In the meantime I found another solution by playing around with custom build systems and got it to work with by using…

```json
"cmd": ["/opt/homebrew/bin/pipenv", "run", "python", "-u", "$file"]
```

…only to find out that I can't access cameras through Sublime.
